test-utils/src/main/java/io/littlehorse/test/WfRunVerifier.java

```diff
@@ -72,7 +73,8 @@ public WfRunVerifier thenVerifyTaskRunResult(
         return this;
     }
 
-    public WfRunVerifier thenVerifyAllTaskRuns(int threadRunNumber, Consumer<List<TaskRun>> verifier) {
+    public WfRunVerifier thenVerifyAllTaskRuns(Consumer<List<TaskRun>> verifier) {
+        steps.add(new VerifyAllTaskRunsStep(verifier, steps.size() + 1));
```

Comment:
This was very broken...when we tried to add the step it would accept it and then  silently not check it.